docs/api-reference/next/image.md

```diff
@@ -455,6 +455,12 @@ module.exports = {
 }
 ```
 
+### Animated Images
+
+The default [loader](#loader) will automatically bypass Image Optimization for animated images and serve it as-is, since converting all the frames of an animated image on-demand is too slow.
+
+Auto-detection for animated files is best-effort and supports GIF, APNG, and WebP. If you want to explicitly bypass Image Optimization for a given animated image, use the [unoptimized](#unoptimized) prop.
```

Comment:
What does best effort mean here? Does it look at the file extension? How would it fail?

Reply:
The buffer is analyzed using [is-animated](https://www.npmjs.com/package/is-animated) package

Reply:
This is interesting, i didnt know that we dont optimize animated images. So a site with lots of GIFs wont benefit from our optimizations

Reply:
In most cases, animated images are not source images. Instead, they are typically exported from some other source such as a video. Most tools that export to animated gif offer a compression option so theres very few cases where a user would even want to optimize an animated image anyway (so far no one has requested it).